src/main/java/net/minecraftforge/common/BiomeManager.java

```diff
@@ -35,11 +36,11 @@
 public class BiomeManager
 {
     private static TrackedList<BiomeEntry>[] biomes = setupBiomes();
-
+    @Deprecated //Don't call this field, use the proper methods in this class to add and get ocean biomes
     public static List<Biome> oceanBiomes = new ArrayList<Biome>();
+    private static List<OceanBiomeEntry>[] forgeOceanBiomes = setupOceanBiomes();
 
-    static
-    {
+    static {
```

Comment:
no need to change the formatting of this line.

Reply:
Yup will fix 👍 
Must've autoformatted.